File: Practice/src/coding_basic_training/AreaOf2.js
function solution(arr) {
    var answer = [];
    var checked = false;
    for (let i of arr) {
        if (i == 2) checked = true;
        if (checked) answer.push(i);
    }
    while (answer.length > 0 && answer[answer.length - 1] != 2) {
        answer.pop();
    }
    return checked ? answer : [-1];

    // const from = arr.indexOf(2);
    // const end = arr.lastIndexOf(2);

    // return from === -1 ? [-1] : arr.slice(from, end + 1);
}

File: Practice/src/coding_basic_training/AreaOf2.py
def solution(arr):
    answer = []
    checked = False
    for i in arr:
        if i == 2: checked = True
        if checked: answer.append(i)
    while len(answer) > 0 and answer[-1] != 2:
        answer.pop()
    return answer if checked else [-1]
    # if 2 not in arr:
    #     return [-1]
    # return arr[arr.index(2) : len(arr) - arr[::-1].index(2)]
